tests/system/test_signals.c

```diff
@@ -45,6 +57,17 @@ static void child_generate_sigfpe(void *unused)
    volatile int zero_val = 0;
    volatile int val = 35 / zero_val;
 
+#if defined(__riscv)
+/*
+ * RISC-V do not generate exception for zero division operation,
```

Comment:
I have looked at the linux code and there is no SIGFPE signal on the riscv architecture.
If this test needs to be skipped, do I need to use a macro switch in cmd_table.h to turn it off

Reply:
Well, it will be enough to check the architecture in the actual test entry point, `cmd_sigfpe()` and just skip the test with a simple print statement and return 0.

Reply:
fixed!